v1/src/lib.rs

```diff
@@ -275,7 +301,9 @@ impl Volume for InfluxDbVolume {
         };
 
         // Check if the database exists (using storages credentials)
-        if !is_db_existing(&client, &db).await? {
+        let client_clone = client.clone();
```

Comment:
Maybe I missed something. Is there any reason we need to clone here?

Reply:
we absolutely do not ! i forgot them from when i was experimenting with `await_task` 
in `await_task` we needed ownership over client and db. 
Thanks for spotting, removed !